docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -254,6 +254,8 @@ Json Parameters:
            + `container_path` to create a new volume for the container
            + `host_path:container_path` to bind-mount a host path into the container
            + `host_path:container_path:ro` to make the bind-mount read-only inside the container.
+           + `volume_name:container_path` to bind-mount a volume managed by a volume plugin into the container.
+           + `volume_name:container_path` to make the bind mount read-only inside the container.
```

Comment:
I think you're missing the `:ro` ?

and what about the selinux `z` and `Z` modifiers? do they have any affect on volume plugins?